### PR TITLE
PostgreSQL 14 Support

### PR DESCRIPTION
**Pull request by joshraker**

### CODE REVIEW

**Comment by bayley-aptible** [comment]
Looks like tests are failing for a bunch of versions but the tests for 14 didn't run, needs a TAG in `docker-postgresql/.travis.yml`.  I didn't investigate much why tests are failing for most other images, with warning `No PostgreSQL clusters exist` for TAG=13.

File: test-ssl.sh [hunk 1]
```diff
@@ -66,7 +66,7 @@ DB_URL="$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' "$PG_CONTAI
 
 test_accept_cipher --tlsv1_2 "$DB_URL"
 
-if [[ "$IMG" =~ postgresql:13 ]]; then
+if [[ "$IMG" =~ postgresql:1[3-4] ]]; then
```
Comment (bayley-aptible) [inline test-ssl.sh:69]: For readability I'd prefer:

```suggestion
if [[ "$IMG" =~ postgresql:(13|14) ]]; then
```

**Comment by joshraker** [comment]
`9.5-contrib` is failing to compile one of its extensions. @UserNotFound what's your take on removing 9.5 images from the build matrix, like we did with 9.4, since it's no longer receiving updates?

**Comment by UserNotFound** [comment]
> `9.5-contrib` is failing to compile one of its extensions. @UserNotFound what's your take on removing 9.5 images from the build matrix, like we did with 9.4, since it's no longer receiving updates?

I think this is the correct thing to do. The version 9.5 is one year past EOL from Postgres, which is when our tentative version support policy says Aptible will no longer provide that version since it will never be patched, and over time becomes increasingly difficult for us to make changes or even rebuild the image, as you found.

**Comment by joshraker** [comment]
@bayley-aptible fixed broken checks.

**Review by bayley-aptible — APPROVED** [review]

**Pull request merged**